util/update_bundled_ca_certificates.rb

```diff
@@ -4,6 +4,7 @@
 
 URIS = [
   URI('https://rubygems.org'),
+  URI('https://index.rubygems.org'),
   URI('https://staging.rubygems.org'),
   URI('https://s3.amazonaws.com'),
```

Comment:
We don't need to test against S3 anymore, I don't think? If we're connecting directly to S3 from RubyGems, something has gone horribly wrong :grimacing: